examples/lib/defaultHtmlStepUi.js

```diff
@@ -128,8 +128,7 @@ function DefaultHtmlStepUi(_sequencer, options) {
 
       
     }
-
-    if (step.name != "load-image") {
```

Comment:
Deleting the if condition here would result in delete option also available to load-image step which should not be the case.
Instead what you can do is simply pull the insert step button functionality out of that if condition and check if the number of steps involved is 1 then you can disable the button on the current step.
Does that make sense?

Reply:
Shall we take the next step here? Thanks so much!